widgyts/dataset_viewer.py

```diff
@@ -185,16 +185,16 @@ def _mirror_positions():
                 camera_action_box.children[0].stop()
             camera_clip = pythreejs.AnimationClip(
                 tracks=[
-                    pythreejs.QuaternionKeyframeTrack(
-                        ".quaternion",
-                        values=[_["quaternion"] for _ in self.position_list],
-                        times=times,
-                    ),
                     pythreejs.VectorKeyframeTrack(
                         ".position",
                         values=[_["position"] for _ in self.position_list],
                         times=times,
                     ),
+                    pythreejs.QuaternionKeyframeTrack(
```

Comment:
any idea why changing the ordering fixes it? maybe an operation modifying values in place or something? We should just merge if it works, was just curious if you know why :)